test/unit-test/mqtt_agent_utest.c

```diff
@@ -313,6 +313,20 @@ void test_MQTTAgent_ResumeSession_session_present_no_resent_publishes( void )
     TEST_ASSERT_EQUAL( MQTTSuccess, mqttStatus );
 }
 
+void test_MQTTAgent_ResumeSession_session_present_no_puback_found( void )
```

Comment:
I'd prefer something like
```suggestion
void test_MQTTAgent_ResumeSession_session_present_no_publish_found( void )
```
since it's the record of the publish we're storing, not the puback

Reply:
Resolved